### ISSUE TITLE
Bug: Can't click through to UI in Linux Mint

### ISSUE THREAD

**Issue by daiyi** [phw/peek#99]
Hello! I can't click through to the UI while recording on Mint ): I'm running:

Linux Mint 18
Cinnamon 3.0.7

**Comment by Prior99** [comment]
I am experiencing the same problem on Arch Linux using i3.

**Comment by phw** [comment]
@Prior99 For i3 this is a known limitation, see https://github.com/phw/peek/issues/52

**Comment by phw** [comment]
@daiyi: Cannot reproduce on Cinnamon 3.2.6 (Mint 8.1). So I think this was actually a bug that has been fixed.

But while testing on Cinnamon I found some issues with displaying the notifications. Does Peek show a desktop notification for you, after you saved a file? The option "Open file manager after saving" needs to be disabled for this.

For me the notification showed up once after much testing, but only very quickly and never again. Tried different approaches displaying the notification without luck.

**Comment by Prior99** [comment]
@phw sad, but thanks.

**Comment by daiyi** [comment]
@phw oh nice! In the next release? I'm using `peek 0.9.1`

I am indeed getting a notification when "Open file manager after saving" is off.

**Comment by phw** [comment]
@daiyi: No, sorry. I meant this seems to be fixed in Mint 8.1 with Cinnamon 3.2.6. I actually think there is nothing I can do in Peek to fix this. I have so gar only tested on Mint 8.1, but I will also test on Mint 8 to be sure.

**Comment by phw** [comment]
@daiyi: I finally could test this on Mint 18 with Cinnamon 3.0.7, but I cannot reproduce this at all.

Here is an animation of me recording on Cinammon. I have no trouble clicking the underlying UI;

![peek 2017-03-14 18-24](https://cloud.githubusercontent.com/assets/29852/23913498/7e2af596-08e3-11e7-9330-65e72289b42f.gif)

Not sure what this could be. What video driver are you using? Do you use the 2D mode of Cinnamon (which I tested also, and works for me too).

**Comment by daiyi** [comment]
This is my `inxi -G` info:
```
Graphics:  Card-1: Intel 3rd Gen Core processor Graphics Controller
           Card-2: NVIDIA GF117M [GeForce 610M/710M/810M/820M / GT 620M/625M/630M/720M]
           Display Server: X.Org 1.18.3 drivers: intel (unloaded: fbdev,vesa) FAILED: nouveau
           Resolution: 1920x1080@60.00hz
           GLX Renderer: Mesa DRI Intel Ivybridge Mobile
           GLX Version: 3.0 Mesa 11.2.0
```

I can't find information about Cinnamon's 2D mode. What is that? How can I tell if I'm using it?

I'm now using peek 1.0.3 and I still can't click through ):

![peek 2017-08-25 10-32](https://user-images.githubusercontent.com/1589186/29706033-b6d32760-8980-11e7-81fc-688f557eb075.gif)

**Comment by vercotux** [comment]
Having this same issue on 3.6.7 every couple of days. The only way around it is to restart cinnamon (or X).

**Comment by luzpaz** [comment]
Is this still reprucible in `1.3.1` ?

**Comment by fvictorio** [comment]
This still happens to me in Mint with Peek 1.4.0.

**Comment by nicobao** [comment]
Still happens in Fedora release 31.
